### DISCUSSION TITLE
Ability to donate Modrinth payouts back to payout pool

### DISCUSSION THREAD

**Discussion by btwonion** [modrinth/.github#98] [Feature requests]
It'd nice to add the ability of donating your creator revenue to Modrinth. I think for the creators, who earn so little that it is not worth paying out, this feature would be nice.

**Comment by Arbee4ever** [comment]
You can configure the payouts in a way, so everything goes to modrinth

**Comment by btwonion** [comment]
really? where can i change it?

**Comment by Arbee4ever** [comment]
In the Project settings under "Members", when you expand the user on the right, set "Monetization weight" to 0... That should be it afaik

**Comment by btwonion** [comment]
okay. thank you!

**Comment by btwonion** [comment]
Nevertheless there should be a feature in the revenue dashboard

**Comment by triphora** [comment]
This feature more likely to be implemented as a "return amount to payout pool" so that it can be redistributed to other authors when payouts are run the following day. Thus, I've changed the title accordingly.

*P.S.: Please use the "Write a reply" button, don't create a new thread for each individual reply*
